README.md

```diff
@@ -27,12 +27,21 @@ To install the requirements first run the command
 ```
 sudo python3 setup.py install
 ```
+>Note: We only support Python 3.5
 
 >Note: there is no sudo for windows as well as when you have root privelages.
 Just run commands without sudo
 
 >Note: some setups use python instead of python3 and pip3 instead of pip
 
+### For Debian based Linux systems:
+
+Run these commands before running the above command:
+```
+sudo apt-get install -y python3-gi python3-gi-cairo libglib2.0-dev libffi-dev libgirepository1.0 libcairo2-dev python3-cairo-dev
```

Comment:
I tried to find a good source with instructions to install pygobject, but no luck. Ultimately, what I did was install every package that the setup of pygobject or UIP was asking for. That is what the install instructions do.
For non Debian users, I haven't found any source and I cannot test as well, since I don't have other ISOs to run in virtualBox. 
Can we can add instructions for Debian systems for now and open additional issues for instructions for other OSes.